apps/public-docsite/src/pages/Controls/ControlsAreaPage.tsx

```diff
@@ -9,6 +9,13 @@ export interface IControlsPageProps extends IPageProps<Platforms> {}
 
 const apiRequireContext = require.context('@fluentui/public-docsite-resources/dist/api/', true, /^(?!references).*/);
 
+const webPlatformBanner = {
+  banner: {
+    title: 'Fluent UI v9',
+    message: 'Check out the all new [Fluent UI version 9](https://react.fluentui.dev)!',
+  },
+};
+
```

Comment:
Sadly, unless we want to repeat this definition in every component doc file and change the type for the IDocPageProps. this is the easiest way to add the banner to every component. If y'all prefer repeating it to avoid this, we can do that.